demos/demo_styled_inputs.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>CSS Chassis - Text Inputs</title>
+	<meta name="description" content="Text inputs different states styling">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="../dist/css/chassis.css">
+	<link rel="stylesheet" href="demos.css">
+	<link href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700,400italic,700italic" rel="stylesheet">
+</head>
+	<body>
+		<div class="demo_container">
+			<h1>CSS Chassis</h1>
+			<div class="demo_container__inputs">
+				<input type="text" placeholder="Name">
```

Comment:
Demos with inputs should show labels associated with them to help enforce best practices, since placeholders should not be used as a label